models/base/scratch/bigquery/snowplow_media_player_base_events_this_run.sql

```diff
@@ -19,24 +19,142 @@ with prep as (
 
   select
 
+    a.app_id,
+    a.platform,
+    a.etl_tstamp,
```

Comment:
You should just be able to do a select * except (or maybe exclude,... whatever bigquery one is) which then has the benefit of all context/sde columns being available for people doing custom models as well?